gems/bootstrap/CVE-2018-14040.yml

```diff
@@ -10,12 +11,16 @@ description: |
   (CVE-2018-14041), data-container property of tooltip (CVE-2018-14042)
 cvss_v2: 4.3
 cvss_v3: 6.1
+unaffected_versions:
+  - "< 4.0.0"
```

Comment:
Are we really sure that versions before `4.0.0` are not affected? I can't seem to find information that confirms this.

Reply:
Removed